apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html

```diff
@@ -1505,7 +1504,10 @@
                         <div class="row">
 
                             <div class="form-group col-xs-4">
-                                <input type="text" #shippingDate="ngModel" placeholder="Ship Date" bsDatepicker name="shippingDate" [(ngModel)]="purchaseOrder.templateDetails.other.shippingDate" class="form-control" [bsConfig]="{ dateInputFormat: giddhDateFormat }" [class.invalid-inp]="shippingDate.errors && (shippingDate.dirty || shippingDate.touched)" />
+                                <giddh-datepicker #shippingDate="ngModel" placeholder="Ship Date" [placeholder]="templatePlaceholder?.shippedDateLabel"
```

Comment:
2 placeholders are used. remove 1 of them.

Reply:
which one should I remove ?